guard/src/commands/validate.rs

```diff
@@ -410,7 +410,7 @@ or rules files.
                         data: data_files,
                         output: output_type,
                         writer,
-                        exit_code: 0,
```

Comment:
is this initialized to 0 in the new code?

Reply:
Yes